rts/lra/lra-test/tck/pom.xml

```diff
@@ -43,6 +43,8 @@
                         <exclude>**/TckInvalidParticipantSignaturesTests.java</exclude>
                         <exclude>**/TckParticipantTests.java</exclude>
                         <exclude>**/TckInvalidSignaturesTests.java</exclude>
+                        <!-- https://github.com/eclipse/microprofile-lra/pull/192 -->
+                        <exclude>**/TckTests.java</exclude>
```

Comment:
We need to run the main test suite to validate that our implementation is working (we need to fix the TCK if it is wrong, not disable it). Is the issue that we need to update which TCK build artefact we are pulling in?